### ISSUE TITLE
[EN][Item][Priest] Tier 12 4p bonus not showing properly in combat log

### ISSUE THREAD

**Issue by Kaldros** [Atlantiss/BugTracker#7385]
Priest 4p bonus (http://www.wowhead.com/spell=99152/cauterizing-flame) doesn't show up at all in healing done for the healer who did it. However, it does show up in healing taken.

From World of Logs:
Healing Done by the priest
![ctflame](https://cloud.githubusercontent.com/assets/15013668/15095942/d66e93c0-14e2-11e6-8c64-597164c6d9ed.png)
Healing Taken by the same priest
![ctflame2](https://cloud.githubusercontent.com/assets/15013668/15095943/d671f858-14e2-11e6-8b1d-1f77e30760cd.png)
Healing Taken by a another raid member on the same fight
![ctflame3](https://cloud.githubusercontent.com/assets/15013668/15095957/3703b83c-14e3-11e6-8899-59065ac2051e.png)
Cauterizing Flame summary for the whole fight
![ctflame_summary](https://cloud.githubusercontent.com/assets/15013668/15095951/f4b9c796-14e2-11e6-99e6-c8fe2a694e33.png)

**Comment by mati865** [comment]
Have you chcked Skada? WoL is updated for retail and some things don't work properly for older expansions.

**Comment by Kaldros** [comment]
Yes. I've checked in Skada and Recount as well. They don't show it either.

**Comment by Kozakko** [comment]
After a small research i can tell that Cauterizing Flame is quite bugged. On oure Majordomo encounter (4:45 min) it healed the raid for amount of 1kk (http://www.worldoflogs.com/reports/6g2c9b4wz8dgojwa/spell/99152/?s=4240&e=4525) thats 16 % of healing done by our disci priest. Thanks to wowhead comments we know that it should have 45 s icd (like other t12 bonuses) and ability to crit. http://www.wowhead.com/spell=99136/cauterizing-flame#comments

**Comment by Kozakko** [comment]
And WoL from retail fight http://www.worldoflogs.com/reports/gszovbyi4cxemu2p/spell/99152/?s=11292&e=11584

**Comment by Malylolek** [comment]
SET bonus dont have ICD and dont show in recount and skada becouse wrong source of caster, WOL look cauterize fire like pet

**Comment by Aival94** [comment]
Reported here as well: https://github.com/Atlantiss/BugTracker/issues/6234

Furthermore i believe it have been fixed the bug you mention but for some reason [Cauterizing Flame](http://www.wowhead.com/spell=99136/cauterizing-flame) is pulling mobs from too far away(more than 40 yards) which make the healing really bad when you are doing trashes in raids or other dungeons.

**Comment by Jenovas** [comment]
Cauterizing Flame from what I remember makes the priest summon an invisible "pet / guardian" that is called  Cauterizing Flame a does the healing, this pet is level 1 from what I remember and that would cause mobs from very far away to pull.

**Comment by Vyrtuz** [comment]
Confirmed, mobs can even kill Cauterizing Flame.
(http://imgur.com/a/LhmuL)

**Comment by Aival94** [comment]
After latest fix seems the the bug got fixed(or at least the range decreased by a huge amount).
